lib/ansible/modules/system/filesystem.py

```diff
@@ -39,16 +40,19 @@
     default: 'no'
   resizefs:
     description:
-    - If C(yes), if the block device and filessytem size differ, grow the filesystem into the space.
-    - Note, XFS Will only grow if mounted.
+    - If C(yes), if the block device and filesytem size differ, grow the filesystem into the space.
+    - Supported for C(ext2), C(ext3), C(ext4), C(ext4dev), C(lvm) and C(xfs) filesystems.
+    - XFS Will only grow if mounted.
     type: bool
     default: 'no'
     version_added: "2.0"
   opts:
     description:
     - List of options to be passed to mkfs command.
 notes:
-  - Uses mkfs command.
+  - Uses tools related to the I(fstype) and C(blkid) command. When I(resizefs) is enabled, C(blockdev) command is required too.
```

Comment:
the problem already existed, but this line really belongs in 'requirements' instead of notes section.

Reply:
Done